### ISSUE TITLE
Update the reviews table that it isn't so wide. It probably shouldn't be a table at all.

### ISSUE THREAD

**Issue by sheenarbw** [Umuzi-org/Tilde#139]
FOR SBONELO

Look at this story: src/stories/CardDetailsModal.Reviews.stories.js 

It demonstrates the component that we use for displaying reviews when we are looking at the details of an agile card.  The reviews component looks pretty bad:

- it's very wide
- it wasted a lot of space
- it doesn't look ok on a mobie device 

We need to display the same info in a better way. 

I have an idea. If you have a better idea then please let me know what it is before diving in. 

Idea for layout:
- Have one [Paper](https://material-ui.com/components/paper/) or [Card](https://material-ui.com/components/cards/) per review. 
- Arrange them in a single column 
- they should be as wide as the modal that they live inside
- they should of course be ordered by date

Each review needs to display the usual information:
- status: Is it competent or not yet competent? Something else? You can import the review icons from here src/components/widgets/CardBadges.js 
- date
- reviewer user info
- validated (that's those ticks and crosses that show up, it says if someone disagrees with your review)
- the actual review text

## Procedure:

1. make a new story just for demonstrating a single review card. Make sure you can display different combinations of information. Make sure it looks reasonable if the text is really wide, even when viewed on a mobile device.
2. Make a PR and get it merged
3. Once your PR is merged, edit src/components/regions/CardDetailsModal/Reviews.js so that it no longer draws a table. It should make use of your new review card. This story will help you make sure it looks good src/stories/CardDetailsModal.Reviews.stories.js 
4. Annother seperate PR

## NB branching procedure

First make 1 branch for step 1 above.
When you need to start with Step 3, make a new branch. Branch it off develop.
